CMakeLists.txt

```diff
@@ -34,6 +37,44 @@ set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/executable)
 # Move assets to build directory
 file(COPY ./assets DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 
+
+
+
+option(BUILD_GP_ONLY "Build only gp_main.cpp" OFF)
```

Comment:
I like whitespace, but we should eventually cut some of this down.

Reply:
removed whitespaces